### DISCUSSION TITLE
Unable to debug on android

### DISCUSSION THREAD

**Discussion by Stardusstt** [unoplatform/uno#9577] [General]
I trying to use uno and create a new project.
But debug on android failed , there is no valid device in debug menu.

using VS 17.4.0 Preview 1.0
maui can start android emulator.

1>------ Deploy started: Project: App1.Mobile, Configuration: Debug Any CPU ------
1>Please select a valid device before running the application.
1>Deploy failed
========== Deploy: 0 succeeded, 1 failed, 0 skipped ==========

<img width="629" alt="image" src="https://user-images.githubusercontent.com/40061990/186235401-37257106-403e-4900-93f2-e6c3833d1551.png">

**Comment by jeromelaban** [comment]
It's a known issue, see https://github.com/unoplatform/uno/discussions/9513. You can upvote the issue on the Microsoft dev community site, it'll help.
